cluster/clean.sh

```diff
@@ -45,6 +45,7 @@ for i in ${namespaces[@]}; do
     _kubectl -n ${i} delete clusterrolebinding -l 'kubevirt.io'
     _kubectl -n ${i} delete clusterroles -l 'kubevirt.io'
     _kubectl -n ${i} delete serviceaccounts -l 'kubevirt.io'
+    _kubectl -n ${i} delete ovm -l 'kubevirt.io'
```

Comment:
What I meant is that you have to delete the `offlinevirtualmachine` crd.

```
_kubectl -n ${i} delete crd -l 'kubevirt.io'
_kubectl -n ${i} delete crd offlinevirtualmachine
```
We can remove that line in a few days though. It is just to keep CI happy for now.

You can keep the ovm deletion too, but make sure that you invoke the deletions before you delete the CRDs (otherwise it can't work). However it is not that important since the test-suite cleans up before and behind itself.